verl/workers/fsdp_workers.py

```diff
@@ -573,6 +573,24 @@ def load_checkpoint(self, path, del_local_after_load=True):
         if self._is_offload_param:
             offload_fsdp_param_and_grad(module=self.actor_module_fsdp, offload_grad=self._is_offload_grad)
 
+    @register(dispatch_mode=Dispatch.ONE_TO_ALL, execute_mode=Execute.RANK_ZERO)
+    def push_to_hub(self, repo_id: str, local_path: str, private: bool = False, token: str = None):
```

Comment:
the change here is not necessary right? @vermouth1992 ?

Reply:
Hi @NielsRogge,

We have changed all the checkpoint saver in the training code to only save distributed checkpoints. So, there is no way for us to push to huggingface hub inside the code. We require the user to merge checkpoint after training. Would you please add your code to here https://github.com/volcengine/verl/blob/main/scripts/model_merger.py? This also applies to megatron backend later when we refactor the megatron code. Thanks!

Reply:
In fact, the functionality already exists. https://github.com/volcengine/verl/blob/main/scripts/model_merger.py#L154

Reply:
Ok thanks for the pointer @vermouth1992. However pushing that doesn't add a model card and `verl` tags.

Would it be useful to add this functionality?

Reply:
I'd agree with @vermouth1992 that the merger script is already sufficient. I personally prefer a minimal trainer loop. @NielsRogge your efforts are much appreciated. I'll close this PR for now but future contributions are always welcome!